src/components/application_manager/test/commands/hmi/on_seek_media_clock_timer_notification_test.cc

```diff
@@ -51,20 +52,67 @@ using ::utils::SharedPtr;
 using ::application_manager::commands::MessageSharedPtr;
 using am::commands::MessageSharedPtr;
 
+#define NOT_VC false
+#define NOT_MEDIA false
+#define NOT_NAVI false
+
+namespace {
+const uint32_t kAppId = 2014u;
+}  // namespace
+
 class OnSeekMediaClockTimerNotificationTest
     : public components::commands_test::CommandsTest<
-          CommandsTestMocks::kIsNice> {};
+          CommandsTestMocks::kIsNice> {
+ public:
+  OnSeekMediaClockTimerNotificationTest()
+      : applications_(application_set_, applications_lock_) {
+    app_ = ConfigureApp(&app_ptr_, kAppId, NOT_MEDIA, NOT_NAVI, NOT_VC);
+  }
+
+ protected:
+  am::ApplicationSet application_set_;
+  sync_primitives::Lock applications_lock_;
+  DataAccessor<am::ApplicationSet> applications_;
+  am::ApplicationSharedPtr app_;
+  NiceMock<MockApplication>* app_ptr_;
+
+  am::ApplicationSharedPtr ConfigureApp(NiceMock<MockApplication>** app_mock,
```

Comment:
@indlu Please note that it is preffered to name mock objects with "mock_" prefix so please rename the app_mock variable into mock_app, and I also think you need to add mock_ prefix to app_ptr_.